config.py

```diff
@@ -30,6 +33,9 @@ class Config(object):
     PASSWORD_MIN_LENGTH = 8
     PASSWORD_MAX_LENGTH = 160
 
+    OAUTH_SERVICE_URL = os.getenv('OAUTH_SERVICE_URL')
+    PARTY_SERVICE_URL = os.getenv('PARTY_SERVICE_URL')
```

Comment:
Being a bit picky put response operations is following the pattern of not having 'SERVICE' in the config names. OAUTH_URL, PARTY_URL, SECURE_MESSAGE_URL